enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/LernaeanTestSupport.scala

```diff
@@ -87,7 +87,7 @@ trait LernaeanTestSupport extends CypherTestSupport {
   val newRuntimeMonitor = new NewRuntimeMonitor
 
   private def extractFirstScenario(config: TestConfiguration): TestScenario = {
-    val preferredScenario = Scenarios.CommunityInterpreted
+    val preferredScenario = Scenarios.CommunityInterpreted3_3
```

Comment:
This seems to be unrelated to this work, and also seems incomplete, because the same improvement is not done to EnterpriseInterpreted or Compiled.